app/models/student.rb

```diff
@@ -8,5 +8,6 @@ class Student < ApplicationRecord
   has_many :scholarship_oportunities, through: :scholarships
 
   accepts_nested_attributes_for :contact_information
+  accepts_nested_attributes_for :scholarships
```

Comment:
You can do this in the line above. `accepts_nested_attributes_for :contact_information, :scholarships`